src/main/java/com/zollingpaper/backend/paper/service/PaperService.java

```diff
@@ -0,0 +1,49 @@
+package com.zollingpaper.backend.paper.service;
+
+import com.zollingpaper.backend.board.domain.Board;
+import com.zollingpaper.backend.board.repository.BoardRepository;
+import com.zollingpaper.backend.paper.domain.Paper;
+import com.zollingpaper.backend.paper.dto.PaperDetailResponse;
+import com.zollingpaper.backend.paper.dto.PaperDetailResponses;
+import com.zollingpaper.backend.paper.dto.PaperSaveRequest;
+import com.zollingpaper.backend.paper.repository.PaperRepository;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Service;
+
+@Service
+public class PaperService {
+
+    private final BoardRepository boardRepository;
+    private final PaperRepository paperRepository;
+
+    public PaperService(BoardRepository boardRepository, PaperRepository paperRepository) {
+        this.boardRepository = boardRepository;
+        this.paperRepository = paperRepository;
+    }
+
+    public Long savePaper(PaperSaveRequest request) {
+        Board board = boardRepository.findById(request.boardId()).orElseThrow();
+        Paper paper = new Paper(board, request.name(), request.content());
+        Paper savedPaper = paperRepository.save(paper);
+
+        return savedPaper.getId();
+    }
+
+    public PaperDetailResponse getPaperDetail(Long paperId) {
+        Paper paper = paperRepository.findById(paperId).orElseThrow();
+        return new PaperDetailResponse(paper.getId(), paper.getCreatedAt(), paper.getName(), paper.getContent());
```

Comment:
레코드 내부에서 정적 팩토리 메서드 통해 인스턴스를 생성하는 방법은 어떻게 생각하시나요?
이 방식으로 하면 파이프 라이닝으로 깔끔하게 뺄 수 있을 것 같습니다!

```suggestion
        return boardRepository.findById(boardId)
                .map(BoardDetailResponse::of)
                .orElseThrow();
```